### PR TITLE
🎉👍

### PR DESCRIPTION
**Pull request by thecraftman**
fix axiom url

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,16 +64,16 @@ Docker images are available on [DockerHub][docker].
 
 1. Set the following environment variables:
 
-* `AXIOM_DEPLOYMENT_URL`: URL of the Axiom deployment to use
+* `AXIOM_URL`: **https://cloud.axiom.co**
 * `AXIOM_ACCESS_TOKEN`: **Personal Access** or **Ingest** token. Can be
```
Comment (njpatel) [inline README.md:68]: There's no such thing as `AXIOM_ACCESS_TOKEN`, it's called `AXIOM_TOKEN`: https://github.com/axiomhq/axiom-honeycomb-proxy/blob/main/cmd/axiom-honeycomb-proxy/main.go#L18

File: README.md [hunk 2]
```diff
@@ -64,16 +64,16 @@ Docker images are available on [DockerHub][docker].
...
 is advised to use an Ingest Token with minimal privileges only.
 
-2. Run it: `./axiom-honeycomb-proxy` or using docker:
+1. Run it: `./axiom-honeycomb-proxy` or using docker:
 
 ```shell
 docker run -p3111:3111/tcp \
-  -e=AXIOM_DEPLOYMENT_URL=<AXIOM_DEPLOYMENT_URL> \
+  -e=AXIOM_URL=<https://cloud.axiom.co> \
   -e=AXIOM_ACCESS_TOKEN=<AXIOM_ACCESS_TOKEN> \
```
Comment (njpatel) [inline README.md:77]: Same here

File: README.md [hunk 3]
```diff
@@ -64,16 +64,16 @@ Docker images are available on [DockerHub][docker].
...
+* `AXIOM_URL`: **https://cloud.axiom.co**
 * `AXIOM_ACCESS_TOKEN`: **Personal Access** or **Ingest** token. Can be
 created under `Profile` or `Settings > Ingest Tokens`. For security reasons it
 is advised to use an Ingest Token with minimal privileges only.
 
-2. Run it: `./axiom-honeycomb-proxy` or using docker:
+1. Run it: `./axiom-honeycomb-proxy` or using docker:
 
 ```shell
 docker run -p3111:3111/tcp \
```
[the remark this answers is not in the document]
Comment (thecraftman) [reply README.md:75]: also in the Dockerfile

[the remark this answers is not in the document]
Comment (njpatel) [reply README.md]: indeed

**Comment by thecraftman** [comment]
@njpatel let me know if you want me to make any changes.

**Review by njpatel — DISMISSED** [review]

**Review by lukasmalkmus — APPROVED** [review]

**Pull request merged**